python/src/scipp/plot/plot1d.py

```diff
@@ -0,0 +1,127 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @author Neil Vaytet
+
+# Scipp imports
+from .. import config
+from .controller import PlotController
+from .model1d import PlotModel1d
+from .panel1d import PlotPanel1d
+from .profile import ProfileView
+from .sciplot import SciPlot
+from .view1d import PlotView1d
+
+
+def plot1d(scipp_obj_dict=None,
+           axes=None,
+           errorbars=None,
+           masks={"color": "k"},
+           filename=None,
+           figsize=None,
+           ax=None,
+           mpl_line_params=None,
+           logx=False,
+           logy=False,
+           logxy=False,
+           grid=False,
+           title=None):
```

Comment:
Most of these options seem to be forwarded directly, can we just use `kwargs` (for most, not all) and forward that?